runtime/filetype.vim

```diff
@@ -3186,7 +3186,7 @@ au BufNewFile,BufRead */etc/sensors.d/[^.]*	call s:StarSetf('sensors')
 au BufNewFile,BufRead */etc/logcheck/*.d*/*	call s:StarSetf('logcheck')
 
 " Makefile
-au BufNewFile,BufRead [mM]akefile*		call s:StarSetf('make')
+au BufNewFile,BufRead [mM]akefile*		if expand('<afile>:t') !~ g:ft_ignore_pat | call dist#ft#FTmake() | endif
```

Comment:
I don't know why this exists here, but there is also another 'Makefile' section near line 1509, would you kindly move it there?

Reply:
It seems that `s:StarSetf` stuffs are put around there (I’m not sure whether it is the convention or not).